capitalize_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class CapitalizeDecorator < BaseDecorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
- [ ] Please require base_decorator here also because  CapitalizeDecorator is making use of it. If the file is not required, you will encounter an `uninitialized constant BaseDecorator` error when testing.

```ruby
require './base_decorator.rb'

class CapitalizeDecorator < BaseDecorator
   # content goes here
end
```